internal/os/diskutil/diskutil.go

```diff
@@ -0,0 +1,97 @@
+// +build windows
```

Comment:
Can this be made a part of https://github.com/kubernetes-csi/csi-proxy/blob/master/internal/os/disk/api.go ?